### PR TITLE
add github org directions to onboarding

### LINKED ISSUE #31: automatically add users to org upon their first PR to this repo
just an idea

https://github.com/marketplace/actions/inclusive-organization

if we do this, it might be nice to be able to add a github label of some kind if we don't want them to join the org for whatever reason

### PR DESCRIPTION
**Pull request by aryarm**
I'm marking this as a draft for now. I still need to do #31 first

also, I'd like to explain in the labwebsite doc that users won't need to fork the repository once they become a member of our github org

### CODE REVIEW

File: doc/Onboarding.rst [hunk 1]
```diff
@@ -17,6 +17,10 @@ If you are new to the group, this page has a list of things you should do to get
 
 To request access to any of these resources, email gymreklab-help AT googlegroups DOT com and someone will help you out.
 
+You should also add a picture of yourself to `the lab website <https://gymreklab.com/people>`_! Just follow `these directions <labwebsite-add-your-photo>`. Note that this process should automatically give you access to our `Github organization <https://github.com/gymrek-lab>`_, as well.
```
Comment (yli091230) [inline doc/Onboarding.rst]: The `these directions <labwebsite-add-your-photo>` link is not working. Will the final version fix this? Thanks.

Comment (aryarm) [reply doc/Onboarding.rst]: ooh, thanks for catching that

**Comment by aryarm** [comment]
closes #31 

I decided not to make an automatic system, after all. Instead, new lab members can request to be added to the org when they make a PR for their lab photo. The admins of the org will be automatically pinged, since this will require a change to the `people.html` file

**Comment by aryarm** [comment]
@yli091230 , is there any chance you can review this PR again?

**Review by yli091230 — APPROVED** [review]
Looks great! Thanks for the super clear and detailed instructions for updating the lab website!

**Pull request merged**